presto-docs/src/main/sphinx/release/release-0.153.rst

```diff
@@ -6,32 +6,154 @@ General Changes
 -----------
 
-* Add support for pluggable resource group management. A ``Plugin`` can now
-  provide management factories via ``getResourceGroupConfigurationManagerFactories()``
-  and the factory can be enabled via the ``etc/resource-groups.properties``
-  configuration file by setting the ``resource-groups.configuration-manager``
-  property. See the ``presto-resource-group-managers`` plugin for an example.
+* Add support for configuring the HTTP Truststore.
 
 Hive Changes
 ------------
 
+* Fix permissions for new tables when using SQL-standard authorization.
+* Allow certain combinations of queries to be executed in a transaction-ish manner,
+  for example, when dropping a partition and then recreating it. Atomicity is not
+  guaranteed due to fundamental limitations in the design of Hive.
+* Fail queries that attempt to rename partition columns.
+* Add support for ORC bloom filters in predicate push down.
+  This is can be enabled using the ``hive.orc.bloom-filters.enabled``
+  configuration property or the ``orc_bloom_filters_enabled`` session property.
+* Add new optimized RCFile reader.
+  This can be enabled using the ``hive.rcfile-optimized-reader.enabled``
+  configuration property or the ``rcfile_optimized_reader_enabled`` session property.
+* Add support for the Presto ``REAL`` type which corresponds to the Hive ``FLOAT`` type.
 * Add support for ``char(x)`` type.
+* Add support for creating, dropping and renaming schemas (databases).
+  The filesystem location can be specified when creating a schema,
+  which allows, for example, easily creating tables on S3.
+* Record Presto query ID for tables or partitions written by Presto
+  using the ``presto_query_id`` table or partition property.
+* Support per-transaction cache for Hive metastore.
+* Include path name in error message when listing a directory fails.
+* Rename ``allow-all`` authorization method to ``legacy``. This
+  method is deprecated and will be removed in a future release.
+* Do not retry intentionally-aborted S3 requests.
+* Set the user agent suffix for S3 requests to ``presto``.
+* Allow configuring the user agent prefix for S3 requests
+  using the ``hive.s3.user-agent-prefix`` configuration property.
+* Add support for S3-compatible storage using the ``hive.s3.endpoint``
+  and ``hive.s3.signer-type`` configuration properties.
+* Add support for using AWS KMS with S3 as an encryption materials provider
+  using the ``hive.s3.kms-key-id`` configuration property.
+* Allow configuring a custom S3 encryption materials provider using the
+  ``hive.s3.encryption-materials-provider`` configuration property.
 
-PostgreSQL Changes
-------------------
+JMX Changes
+-----------
 
-* Add support for ``char(x)`` type.
+* Make name configuration for history tables case-insensitive.
 
 MySQL Changes
 -------------
 
-* Add support for ``char(x)`` type.
+* Optimize fetching column names when describing a single table.
+* Add support for ``char(x)`` and ``REAL`` data types.
```

Comment:
capitalize "char" or lower-case "real". Also, name it "char(n)", not "char(x)" -- it's more commonly used that way.